timur/lib/client/jsx/actions/view_actions.js

```diff
@@ -10,17 +10,17 @@ const addView = (view_name, view) => ({ type: 'ADD_VIEW', view_name, view })
  * Request a view for a given model/record/tab and send requests for additional 
  * data.
  */
-export const requestView = (model_name, success, error) => (dispatch) => {
+export const requestView = (model_name) => (dispatch) => {
```

Comment:
This was used in only one place so that change was easy enough.  The callbacks are no longer necessary since `invoke` will both pass through the dispatch for us and give back the return value (a promise).